charts/workflows/charts/move2kube/templates/01-sonataflow_m2k.yaml

```diff
@@ -255,11 +255,15 @@ spec:
   podTemplate:
     container:
       resources: {}
-      image: quay.io/orchestrator/serverless-workflow-move2kube:4fae4bd85a48f4091f5c67a83fbcc94dcd29f1f2
+      image: {{ .Values.workflow.image }}
+      #imagePullPolicy: Always
+      env:
+        - name: K_SINK
+          value: "http://broker-ingress.knative-eventing.svc.cluster.local/{{ .Values.namespace }}/{{ .Values.brokerName }}"
   resources:
     configMaps:
       - configMap:
-          name: 01-m2k-resources
+          name: 01-{{ .Values.workflow.name }}-resources
```

Comment:
same as first comment

Reply:
I am using the `.Values.workflow.name` in the knative resource file as well to target the service in the trigger (subscriber)